<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>medInfo</class>
 <widget class="QWidget" name="medInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>药品管理</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>药品名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="med_name_input"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>厂家</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="supplier_name_input"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_reset">
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_modify">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_del">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="btn_all">
          <property name="text">
           <string>全部</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="type_woman">
          <property name="text">
           <string>妇科用药</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="type_man">
          <property name="text">
           <string>男科用药</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="type_kid">
          <property name="text">
           <string>儿科用药</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="type_nerve">
          <property name="text">
           <string>神经系统用药</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="type_tumour">
          <property name="text">
           <string>抗肿瘤类药物</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="type_skin">
          <property name="text">
           <string>皮肤用药</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="type_eye">
          <property name="text">
           <string>眼科用药</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="type_hot">
          <property name="text">
           <string>治解热镇痛药物</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
